compiler/src/dotty/tools/dotc/parsing/Parsers.scala

```diff
@@ -278,7 +278,7 @@ object Parsers {
       if (in.token == EOF) incompleteInputError(msg)
       else
         syntaxError(msg, offset)
-        skip()
+        skip(stopAtComma = true)
```

Comment:
This line makes it so we skip to the next argument if we encounter an error in a comma separated list.